src/read_sad_lut.F90

```diff
@@ -17,7 +17,8 @@
 ! 2021/03/08, AP: Gather grid dimensions into LUT_Grid_t
```

Comment:
I've made multiple changes to this file. Details here: https://github.com/ORAC-CC/orac/pull/61